README.md

```diff
@@ -940,6 +940,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 - ### **H**
 
   - [Haafizz Ismail](https://github.com/haafizzismail)
+  - [Hossain Kabir](https://github.com/ho-ssain)
```

Comment:
Please add your name in line 994, after Honey Patel.
Let us know if you need any help with that. 🙂